From c34146678d7b54c02f209acde255afde575f9e63 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 25 Nov 2018 11:04:32 +0200
Subject: [PATCH] Makefile: specify clean as .PHONY target

clean is not a file but a recipe. Specify it as .PHONY.

Change-Id: I1b79d7b226a729d31d4b146a221ee7da96f73448
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 15cd6c7..b2751c1 100644
--- a/Makefile
+++ b/Makefile
@@ -45,3 +45,5 @@
 
 clean:
 	rm -f $(PREBOOT_OBJECTS)
+
+.PHONY: clean
\ No newline at end of file
